include/z64cutscene.h

```diff
@@ -22,6 +22,7 @@ typedef enum CutsceneState {
 } CutsceneState;
 
 typedef enum CutsceneCmd {
+    /* 0xFFFF */ CS_CMD_END_OF_SCRIPT = -1,
     /* 0x0001 */ CS_CMD_CAM_EYE_SPLINE = 0x01,
```

Comment:
fwiw the point of the 0 enum value was so that `CS_CMD_CAM_EYE_SPLINE` wouldnt need to have its value explicitly set. It would just increment by one naturally like enums normally do.

But, since people dont want it, I would at least change the 0x01 to 1.

Reply:
I'm not sure about changing it to decimal given the unimplemented commands using hex in their name like `CS_CMD_UNIMPLEMENTED_15`. For consistency I think they should all be hex or decimal, with the exception of this one negative number.

Reply:
Fair enough. Though that reminds me, if the enum already has "unimplemented" to fill gaps in the enum, we should just do that here instead of naming it "none" like I originally suggested.
```
typedef enum CutsceneCmd {
    /* 0xFFFF */ CS_CMD_END_OF_SCRIPT = -1,
    /* 0x0000 */ CS_CMD_UNIMPLEMENTED_0, 
    /* 0x0001 */ CS_CMD_CAM_EYE_SPLINE,
    ```

Reply:
disregard, the unimplemented ones do get used and are not just there to fill gaps. sorry.

Reply:
The unimplemented commands aren’t filling all the gaps. `1B` that I mentioned comes after a missing `1A` command. I’m not sure what version needs these (if any). 1.0 did not show any I searched for in assets.

Reply:
ok yeah im a bit confused why they are there currently. And im the one that did this a couple years ago 😅 but anyway we can just disregard for the sake of this pr